.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,71 @@
+name: Bug report
+description: Create a report to help us improve
+title: '[User reported bug]: '
+labels: C-bug, S-needs-triage
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+      value: "I expected to see this happen:
+
+
+      Instead, this happened:
+
+    "
+    validations:
+      required: true
+  - type: textarea
+    id: reproduce
+    attributes:
+      label:  What were you doing when the issue happened?
+      description: Copy and paste the exact commands or code here.
+      placeholder: "Behavior or code sample that causes the bug"
+    validations:
+      required: false
+  - type: textarea
+    id: logs
+    attributes:
+      label: Zebra logs
+      description: Copy and paste the last 100 Zebra log lines or upload the full logs to https://gist.github.com/ and add a link to them here.
+      placeholder: "Copy and paste the logs here"
+    validations:
+      required: false
+  - type: input
+    id: zebrad-version
+    attributes:
+      label: Zebra Version
+      description: "For bugs in `zebrad`, run `zebrad --version`."
+      placeholder: "zebrad 1.0.0-rc.9"
```

Some users will copy this text, so it shouldn't be a real version

```suggestion
      placeholder: "zebrad 1.0.0-placeholder"
```